test/integration/redirect_integration_test.cc

```diff
@@ -96,6 +96,7 @@ TEST_P(RedirectIntegrationTest, RedirectNotConfigured) {
   auto response = sendRequestAndWaitForResponse(default_request_headers_, 0, redirect_response_, 0);
   EXPECT_TRUE(response->complete());
   EXPECT_EQ("302", response->headers().getStatusValue());
+  EXPECT_EQ(1, test_server_->counter("http.config_test.downstream_rq_3xx")->value());
```

Comment:
Please verify that the right response code makes it to the access log.

Add to the top of the test:
useAccessLog("%RESPONSE_CODE% %RESPONSE_CODE_DETAILS%");

Add to the bottom of the test:
EXPECT_THAT(waitForAccessLog(access_log_name_), HasSubstr(...));
for each of the expected responses: the 3xx redirect and the real response.

Reply:
Done now. For a lot of these tests, the final response code is actually 302.